client/webserver/site/src/js/markets.ts

```diff
@@ -1008,6 +1021,68 @@ export default class MarketsPage extends BasePage {
     } else this.page.submitBttn.textContent = intl.prep(intl.ID_SET_BUTTON_BUY, { asset: Doc.shortSymbol(this.market.baseCfg.unitInfo.conventional.unit) })
   }
 
+  setOrderBttnEnabled (isEnabled: boolean, disabledTooltipMsg?: string) {
+    const btn = this.page.submitBttn
```

Comment:
I recommend putting a line here.

```Javascript
console.error("setOrderBttnEnabled")
```
and fiddling with input values. Try to get this down to running once per action.

Reply:
The above change eliminated superfluous `updateOrderBttnState` calls.